pkg/apis/core/validation/shoot.go

```diff
@@ -1480,6 +1480,9 @@ func ValidateVerticalPodAutoscaler(autoScaler core.VerticalPodAutoscaler, fldPat
 	allErrs = append(allErrs, featuresvalidation.ValidateVpaFeatureGates(autoScaler.FeatureGates, fldPath.Child("featureGates"))...)
 	allErrs = append(allErrs, ValidateVerticalPodAutoscalerMaxAllowed(autoScaler.MaxAllowed, fldPath)...)
 
+	if workerCount := autoScaler.UpdateWorkerCount; workerCount != nil {
+		allErrs = append(allErrs, apivalidation.ValidateNonnegativeField(*workerCount, fldPath.Child("updateWorkerCount"))...)
```

Comment:
Does it make sense to check if the `workerCount > 0`, i.e. it's a positive number, rather than including `0` as an accepted value ?

From what I managed to find, the _upstream_ `vpa-recommender` does not perform any validation on that exact [flag](https://github.com/kubernetes/autoscaler/blob/bc3f44c85df17bccc940adb7c885b192cf6135d7/vertical-pod-autoscaler/pkg/recommender/main.go#L69), so I was wondering if we should enforce _positive number_ inputs ? WDYT ?